crates/objc2/src/runtime/nsobject.rs

```diff
@@ -155,6 +163,36 @@ pub unsafe trait NSObjectProtocol {
     // For example, something may have a return type of `NSString`, while
     // behind the scenes they really return `NSMutableString` and expect it to
     // not be modified.
+
+    /// Checks if the object is a member of the class.
+    ///
+    /// See [Apple's documentation][apple-doc] for more details on what you
+    /// may (and what you may not) do with this information.
+    ///
+    /// [apple-doc]: https://developer.apple.com/documentation/objectivec/1418956-nsobject/1418766-ismemberofclass
+    #[doc(alias = "isMemberOfClass")]
+    fn is_member_of<T: ClassType>(&self) -> bool
```

Comment:
It seems to me like this is not _that_ useful? A faster implementation would basically just be `self.class() == T::class()`, which I also think is more understandable to a user than `is_member_of`/`isMemberOfClass`?

Reply:
I agree.